### PR TITLE
disable viewport scale

### PR DESCRIPTION
**Pull request by missinglink**
temporarily disable viewport scale modifications

### CODE REVIEW

**Comment by dianashk** [comment]
:+1:

File: test/unit/query/search.js [hunk 1]
```diff
@@ -90,22 +90,23 @@ module.exports.tests.query = function(test, common) {
...
-      layers: ['test']
-    });
-
-    var compiled = JSON.parse( JSON.stringify( query ) );
-    var expected = require('../fixture/search_linguistic_viewport_min_diagonal');
-
-    t.deepEqual(compiled, expected, 'valid search query');
-    t.end();
-  });
+  // viewport scale sizing currently diabled.
```
Comment (orangejulius) [inline test/unit/query/search.js]: Ooh, add a link here to https://github.com/pelias/api/issues/387

Comment (missinglink) [inline test/unit/query/search.js]: as if by magic it has appeared!

Comment (orangejulius) [inline test/unit/query/search.js]: I know star wars came out today but force push? :P

Comment (missinglink) [inline test/unit/query/search.js]: pull? there is no pull! only fetch and merge there are!

**Comment by orangejulius** [comment]
Fixes #387

**Pull request merged**